Move SAML to its own dedicated authentication section (#833)
clerk/clerk-docs: docs/custom-flows/saml-connections.mdx

# SAML connections

> In case our default [SAML flow](/docs/authentication/saml/overview) doesn’t cover your needs, you can leverage the Clerk SDK to build a completely custom SAML flow.

Reviewer: ```suggestion
If [the default SAML flow](/docs/authentication/saml/overview) doesn’t cover your needs, you can build a custom SAML flow with the Clerk SDK.
```
